test/onnx/test_lazy_import.py

```diff
@@ -0,0 +1,23 @@
+# Owner(s): ["module: onnx"]
+
+import subprocess
+import sys
+import tempfile
+
+import pytorch_test_common
+
+from torch.testing._internal import common_utils
+
+
+class TestLazyONNXRuntime(pytorch_test_common.ExportTestCase):
+
+    def test_onnxruntime_is_lazily_imported(self):
+
+        with tempfile.TemporaryDirectory() as wd:
+            r = subprocess.run([sys.executable, '-Ximporttime', '-c', 'import sys, torch.onnx'], capture_output=True, text=True, cwd=wd, check=True)
+
+        self.assertTrue(' onnxruntime' not in r.stderr, f'`onnxruntime` should not be imported, full importtime: {r.stderr}')
```

Comment:
```suggestion
        self.assertTrue('onnxruntime' not in r.stderr, f'`onnxruntime` should not be imported, full importtime: {r.stderr}')
```

Is the space intended?

Also this file needs formatting

Reply:
Yes that's a little bit hack but necessary. The space makes sure this test check for the `onnxruntime` library, not
`torch.onnx._internal.onnxruntime`.

Reply:
In that case I would add a comment there